backends-clickhouse/src/main/scala/org/apache/spark/sql/execution/datasources/v1/CHParquetWriterInjects.scala

```diff
@@ -14,39 +14,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.spark.sql.execution.datasources.v1.nativeparquet
+package org.apache.spark.sql.execution.datasources.v1
 
 import io.glutenproject.vectorized.CHColumnVector
 
-import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.catalyst.InternalRow
-import org.apache.spark.sql.execution.datasources.{CHDatasourceJniWrapper, FakeRow, GlutenParquetFileFormat, OutputWriter, OutputWriterFactory}
-import org.apache.spark.sql.execution.datasources.parquet.ParquetUtils
-import org.apache.spark.sql.sources.DataSourceRegister
+import org.apache.spark.sql.execution.datasources._
 import org.apache.spark.sql.types.StructType
 
-import org.apache.hadoop.fs.FileStatus
-import org.apache.hadoop.mapreduce.{Job, TaskAttemptContext}
+import org.apache.hadoop.mapreduce.TaskAttemptContext
 import org.apache.parquet.hadoop.codec.CodecConfig
 
-class CHParquetFileFormat
-  extends GlutenParquetFileFormat
-  with DataSourceRegister
-  with Serializable {
-
-  override def inferSchema(
-      sparkSession: SparkSession,
-      options: Map[String, String],
-      files: Seq[FileStatus]): Option[StructType] = {
-    ParquetUtils.inferSchema(sparkSession, options, files)
-  }
-
-  override def prepareWrite(
-      sparkSession: SparkSession,
-      job: Job,
-      options: Map[String, String],
-      dataSchema: StructType): OutputWriterFactory = {
+class CHParquetWriterInjects extends GlutenParquetWriterInjectsBase {
 
+  def createOutputWriterFactory: OutputWriterFactory = {
```

Comment:
`HiveFileFormat `and `ParquetFileformat `both call this method to create OutputWriterFactory? The filename of `HiveFileFormat  `and `ParquetFileformat  `is different. [PR#2328](https://github.com/oap-project/gluten/pull/2328) update the filename of `Hiveformat `to align with vanilla spark.

Reply:
our implementation does not involve naming of the file.  the file name is passed in as a parameter. correct me again if I understand it wrong

Reply:
VeloxParquetFileFormat generate the filename in [here](https://github.com/oap-project/gluten/blob/55103a3e113c597572c6c552fd090a73ac6d5da9/backends-velox/src/main/scala/org/apache/spark/sql/execution/datasources/velox/VeloxParquetFileFormat.scala#L66). And the HiveFileFormat code is [here](https://github.com/oap-project/gluten/blob/55103a3e113c597572c6c552fd090a73ac6d5da9/backends-velox/src/main/scala/org/apache/spark/sql/hive/execution/HiveFileFormat.scala#L138).